### PR TITLE
SJSU-3001,  Bugs in Individual Hotel Page

### PR DESCRIPTION
**Pull request by itjinshan**
All bugs are fixed base on the story descriptions except changing the price to decimal places. 

**Mobile Compatible Let's Go!!!**

bitmoji

### CODE REVIEW

**Comment by SandroSal** [comment]
lets goooo

**Comment by SandroSal** [comment]
please call Search container something like "modify search"

**Comment by itjinshan** [comment]
Thank you @SandroSal for the feedback.

I have changed most things beside room price, I still want to leave it as Integer since most major hotel booking sites are using integers.
And I think the map is longer than pictures creates a more dynamic user experience, all the information transit more smoothly. Lining up map and pictures will make the whole page look more section isolated. It does make the page more organized, but I like my design more.

bitmoji

File: front_end/src/actions/searchResultActions.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { GET_INDIVIDUAL_HOTEL } from "./types";
+import { GET_INDIVIDUAL_HOTEL, CLEAR_DATA } from "./types";
```
Comment (SandroSal) [inline front_end/src/actions/searchResultActions.js]: why do we clear the data?

File: front_end/src/components/history-search/HistorySeach.js [hunk 2]
```diff
@@ -0,0 +1,508 @@
+import React, { Component } from "react";
```
Comment (SandroSal) [inline front_end/src/components/history-search/HistorySeach.js]: please create separate PR for history

File: front_end/src/components/indiv-hotel/IndivHotel.js [hunk 3]
```diff
@@ -6,11 +6,36 @@ import { connect } from "react-redux";
...
+
+import "./indivHotel.css";
 import taxrates from "../payment/taxrates.json";
 
 class IndivHotel extends Component {
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
+
+  componentWillUnmount(){
```
Comment (SandroSal) [inline front_end/src/components/indiv-hotel/IndivHotel.js]: if we clear the data, we won't be able to use the browsers back button anymore...

File: front_end/src/components/indiv-hotel/indivHotel.css [hunk 4]
```diff
@@ -0,0 +1,737 @@
+@media screen and (max-width: 330px){
+  .wholePage{
+    margin-left: 4%;
+    margin-right: 4%;
+  }
+
+    .seeRoomBtn {
+      width: 100%;
+      /* variant="contained" */
```
Comment (SandroSal) [inline front_end/src/components/indiv-hotel/indivHotel.css]: please go through css and take out all the commented out code

File: front_end/src/reducers/index.js [hunk 5]
```diff
@@ -14,5 +14,5 @@ export default combineReducers({
   profile: profileReducer,
   landing: landingReducer,
   individualHotelData: individualHotelReducer,
-  bookingData: bookingReducer
+  bookingData: bookingReducer,
```
Comment (SandroSal) [inline front_end/src/reducers/index.js]: please undo

File: front_end/src/App.js [hunk 6]
```diff
@@ -74,7 +74,6 @@ class App extends Component {
             </StripeProvider>
             <Route exact path="/confirmation" component={Confirmation} />
             <Route exact path="/aboutus" component={AboutUs} />
```
Comment (SandroSal) [inline front_end/src/App.js]: please undo

**Comment by itjinshan** [comment]
Fixed and cleaned up the codes.

**update**
Added a section for reviews.
search for "InterContinental Boston" with any dates to see reviews

**Pull request merged**